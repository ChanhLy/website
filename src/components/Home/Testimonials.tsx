export interface TestimonialsProps {
  className?: string;
}

export const Testimonials: React.FC<TestimonialsProps> = (props) => {
  return (
    <div className="bg-blue-50" {...props}>
      <div className="	 text-gray-900 flex justify-between ">
        {/* {AppConfig.title} */}
        <div className="block text-center my-7">
          <button className="rounded-full	bg-gray-300">IMG</button>
          <p className="my-5">
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
          <h1 className="font-bold my-5">Your title</h1>
        </div>

        <div className="block text-center my-7">
          <button className="rounded-full	bg-gray-300">IMG</button>
          <p className="my-5">
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
          <h1 className="font-bold my-5">Your title</h1>
        </div>

        <div className="block text-center my-5">
          <button className="rounded-full	bg-gray-300">IMG</button>
          <p className="my-5">
            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </p>
          <h1 className="font-bold my-5">Your title</h1>
        </div>
      </div>
    </div>
  );
};
