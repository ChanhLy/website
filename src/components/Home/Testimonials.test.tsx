import React from 'react';

import renderer from 'react-test-renderer';

import { Testimonials } from './Testimonials';

it('renders correctly', () => {
  const tree = renderer.create(<Testimonials></Testimonials>).toJSON();
  expect(tree).toMatchSnapshot();
});
