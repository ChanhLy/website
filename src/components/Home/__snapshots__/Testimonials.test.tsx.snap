// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-blue-50"
>
  <div
    className="	 text-gray-900 flex justify-between "
  >
    <div
      className="block text-center my-7"
    >
      <button
        className="rounded-full	bg-gray-300"
      >
        IMG
      </button>
      <p
        className="my-5"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      </p>
      <h1
        className="font-bold my-5"
      >
        Your title
      </h1>
    </div>
    <div
      className="block text-center my-7"
    >
      <button
        className="rounded-full	bg-gray-300"
      >
        IMG
      </button>
      <p
        className="my-5"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      </p>
      <h1
        className="font-bold my-5"
      >
        Your title
      </h1>
    </div>
    <div
      className="block text-center my-5"
    >
      <button
        className="rounded-full	bg-gray-300"
      >
        IMG
      </button>
      <p
        className="my-5"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      </p>
      <h1
        className="font-bold my-5"
      >
        Your title
      </h1>
    </div>
  </div>
</div>
`;
